test_data/strain/imperfect_solution/expected_analysis.json

```diff
@@ -0,0 +1,2 @@
+{"summary":"Check the comments for things to fix. 🛠 ","comments":[{"comment":"elm.strain.do_not_use_filter","type":"essential","params":{}}]}
```

Comment:
It's important to have https://github.com/exercism/website-copy/pull/2161/files merged before the `analyzer` property is set to `true` in the track's `config.json` file: https://github.com/exercism/elm/blob/main/config.json#L19

Reply:
Noted! I didn't know about that line in`config.json`, thank you.

I don't think I can request a review from @mpizenberg or @ceddlyburge on website-copy, they are not on the list. How can we add them in there? We will need to approve analyzer comments in the future. Maybe in the mean time you could approve it?

Reply:
Indeed, I'd prefer not to be added to website-copy. I don't mind occasionally being pingged there for analyzer messages though, even if I cannot officially review. I might just comment if I find the time, or not if I don't.

Reply:
I've just merged the website-copy PR